### PR TITLE
[#818] Replace creating of a tag with a branch in the CI pipeline

### PR DESCRIPTION
**Pull request by mcdoker18**

### CODE REVIEW

**Comment by mcdoker18** [comment]
bot, test infra

**Comment by mcdoker18** [comment]
bot, test infra

**Comment by mcdoker18** [comment]
bot, test infra

**Review by kirillmakhonin — DISMISSED** [review]

**Review by botsman — DISMISSED** [review]

File: deploy/ci.Jenkinsfile [hunk 1]
```diff
@@ -35,15 +35,15 @@ pipeline {
                 cleanWs()
                 checkout scm
                 script {
-                    print('Set interim merge tag')
+                    print('Set interim merge branch')
                     sh """
-                    echo ${env.param_push_git_tag}
-                    if [ `git tag |grep -x ${env.mergeTag}` ]; then
+                    echo ${env.mergeTag}
```
Comment (alexander-semenets) [inline deploy/ci.Jenkinsfile]: shouldn't we rename param as well? 
e.g env.mergeBranch

Comment (mcdoker18) [reply deploy/ci.Jenkinsfile]: Good point! Fixed

**Comment by mcdoker18** [comment]
bot, test infra

**Review by botsman — APPROVED** [review]

**Review by kirillmakhonin — APPROVED** [review]

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**